test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -989,18 +959,18 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
 
 				// Verify cpus are back online when the offline parameters is removed
-				for _, node := range workerRTNodes {
-					numaCoreSiblings, err = nodes.GetCoreSiblings(context.TODO(), &node)
-				}
 				for _, cores := range numaCoreSiblings {
 					for _, cpuSiblings := range cores {
 						for _, cpus := range cpuSiblings {
 							allCpus = append(allCpus, cpus)
 						}
 					}
 				}
+				// we don't need to check all the cpus as this may be costly
+				// on BM system which have more than 100 cpus, so we pick any 10 cpus randomly.
+				cpusToCheck := randomCpus(allCpus)
 				for _, node := range workerRTNodes {
-					for _, v := range allCpus {
+					for _, v := range cpusToCheck {
```

Comment:
You can still check all cpus and even reduce it to single call using:
`find /sys/devices/system/cpu/cpu* -type f -name online -exec cat {} \;`
The check that all lines are equal to `1`

Reply:
addressed it in the latest commit.

Reply:
Or cat `/sys/devices/system/cpu/online` and check the corresponding cpuset. The current method looks good enough, no need for further changes now.